### PR TITLE
add setUniform example

### PR DESCRIPTION
**Pull request by Spongman**
add an example for `p5.Shader.setUniform`.

### CODE REVIEW

File: src/webgl/p5.Shader.js [hunk 1]
```diff
@@ -279,6 +279,36 @@ p5.Shader.prototype.useProgram = function() {
...
+ *   createCanvas(100, 100, WEBGL);
+ *   shader(twist);
+ *   for (var i = 0; i < 8; i++) {
+ *     ps.push({ x: 1 + random(), y: 1 + random() });
+ *   }
+ * }
+ * function draw() {
+ *   var time = millis() / 5000 + 1000;
+ *   for (var i = 0; i < 8; i++) {
+ *     twist.setUniform('v' + i, [sin(time / ps[i].x), cos(time / ps[i].y)]);
```
Comment (lmccart) [inline src/webgl/p5.Shader.js]: can we do this without sin() and cos(), and without the for loop?

File: src/webgl/p5.Shader.js [hunk 2]
```diff
@@ -279,6 +279,36 @@ p5.Shader.prototype.useProgram = function() {
...
+ *   }
+ *
+ *   background(100);
+ *   scale(width / 2, height / 2);
+ *   quad(-1, -1, 1, -1, 1, 1, -1, 1);
+ * }
+ * </code></div>
+ *
+ * @alt
+ * colorful moving lines
```
Comment (lmccart) [inline src/webgl/p5.Shader.js]: this ideally would be more descriptive. imagine a person that can't see the screen, we want to give them some idea of visually what is going on. it's less of a caption, more of a text translation of graphics.

**Comment by kjhollen** [comment]
hi @Spongman, are you planning to update this to account for Lauren's review comments? let me know, and if not and I'll close the PR. thanks!

**Comment by kjhollen** [comment]
Closing because of inactivity. If you'd like to make the requested changes, please reopen. Thanks!

**Pull request closed without merging**